### ISSUE TITLE
Get Linting up and running

### ISSUE THREAD

**Issue by theweekendgeek** [theweekendgeek/jambot#14]
e. g. set up eslint with the correct config

**Comment by theweekendgeek** [comment]
So I could have guesses this but TS-linting was pretty much set up out of the box. I just added a rule, regarding newlines before returns.

Regarding PHP linting... How do you usually handle this usually? I've been thinking about the need for a php executable that most php linters that I've used so far needed!?

**Comment by BobMali** [comment]
I usually use a combination of [PHP Mess Detector](https://phpmd.org/), [PHP Copy Paste Detector](https://github.com/sebastianbergmann/phpcpd) and expecially [PHP Codesniffer](https://github.com/squizlabs/PHP_CodeSniffer). They are all available as composer packages. I have them installed local as global dependencies but we should add them to our project composer dependencies. The integrate quite well into Jetbrains-IDE's.

**Comment by theweekendgeek** [comment]
Yes, sounds good to me. I add them as dependencies and then close this issue. I imagine It will be  a bit of a challenge to get them all to play nice with docker on my machine :)
